package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "imagekit-uppy-plugin",
-  "version": "1.1.0-alpha.1",
+  "version": "1.1.1-alpha.1",
```

Comment:
Since this plugin will not be compatible with < 3.x versions of uppy it should be a major release. `2.0.0-alpha.0`

We can also plan a public release with upload API v2 that uses JWT separately.

Reply:
updated the version name